applications/app/views/fragments/interactiveBody.scala.html

```diff
@@ -62,4 +62,22 @@
         </div>
         @fragments.contentFooter(interactive, page.related, isPaidContent = isPaidContent)
     }
+
+    <script>
+        @HtmlFormat.raw(common.Assets.js.curl)
+
+        [].slice.apply(document.querySelectorAll('figure.interactive')).forEach(function (el) {
+            var mainJS = el.getAttribute('data-interactive');
+            if (!mainJS) {
```

Comment:
would `document.querySelectorAll('figure.interactive[data-interactive]')` simplify this? remove the need to exit?